### PR TITLE
{monitor} fix four testcases

### PR DESCRIPTION
**Pull request by YuanyuanNi**
**Description**

fix testcases   
below testcases are effected:
test_monitor_log_analytics_workspace_saved_search
TestLogProfileScenarios::test_monitor_create_log_profile
TestLogProfileScenarios::test_monitor_log_analytics_workspace_data_export
**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/network/tests/latest/test_network_commands.py [hunk 1]
```diff
@@ -4801,7 +4801,7 @@ def test_network_watcher_configure(self, resource_group):
 class NetworkWatcherScenarioTest(ScenarioTest):
     from unittest import mock
 
-    def _mock_thread_count():
+    def _mock_thread_count(self):
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/tests/latest/test_network_commands.py:4804]: ?

File: src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_analytics_workspace.py [hunk 2]
```diff
@@ -422,7 +424,9 @@ def test_monitor_log_analytics_workspace_data_export(self, resource_group, accou
                      '--destination {sa_id_1} --enable -t "SecurityEvent Heartbeat"',
                      checks=[
                      ])
-        with self.assertRaisesRegex(HttpResponseError, 'you can create 10 export rules to 10 different destinations'):
+        #with self.assertRaisesRegex(HttpResponseError, 'you can create 10 export rules to 10 different destinations'):
+        with self.assertRaisesRegex(HttpResponseError,
+                                        'You are adding a destination that is already defined in rule: clitest. Destination must be unique across export rules in your workspace . See http://aka.ms/LADataExport#limitations'):
```
Comment (wangzelin007) [inline src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_analytics_workspace.py:429]:
```suggestion
                                    'You are adding a destination that is already defined in rule: clitest. Destination must be unique across export rules in your workspace . See http://aka.ms/LADataExport#limitations'):
```
Please pay attention to code alignment and delete the unused code.

File: src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_analytics_workspace.py [hunk 3]
```diff
@@ -432,7 +436,9 @@ def test_monitor_log_analytics_workspace_data_export(self, resource_group, accou
                      '--destination {sa_id_1} --enable -t ABC',
                      checks=[
                      ])
-        with self.assertRaisesRegex(HttpResponseError, 'you can create 10 export rules to 10 different destinations'):
+      #  with self.assertRaisesRegex(HttpResponseError, 'you can create 10 export rules to 10 different destinations'):
+        with self.assertRaisesRegex(HttpResponseError,
+                                        'You are adding a destination that is already defined in rule: clitest. Destination must be unique across export rules in your workspace . See http://aka.ms/LADataExport#limitations'):
```
Comment (wangzelin007) [inline src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_analytics_workspace.py:441]: same as above

**Review by wangzelin007 — CHANGES_REQUESTED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**